CHANGELOG.next.md

```diff
@@ -34,6 +34,32 @@ Thanks, you're awesome :-) -->
 
 #### Deprecated
 
+## 8.3.0 (Soft Feature Freeze)
+
+### Schema Changes
+
+#### Breaking changes
+
+#### Bugfixes
+
+#### Added
+
+#### Improvements
+
+#### Deprecated
+
+### Tooling and Artifact Changes
+
+#### Breaking changes
+
+#### Bugfixes
+
+#### Added
+
+#### Improvements
+
+#### Deprecated
+
 ## 8.2.0 (Soft Feature Freeze)
```

Comment:
We'll want to update the 8.2 for HFF, I believe?

Reply:
Oh maybe it's covered in https://github.com/elastic/ecs/pull/1856

Reply:
Yep, but I did that in a separate PR